### PR TITLE
doc: add some more contributing information

### PR DESCRIPTION
**Pull request by Pixep**
Add high-level information and clarify/update how people can contribute.

### CODE REVIEW

**Comment by Pixep** [comment]
@chandlerc does that fit what you had in mind?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,24 +89,49 @@ eventually implemented. A good starting point is to read through the
...
+marked as
 ["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue).
 These are intended to be a good place to start contributing.
 
+#### Triage, analyze or address bugs
+
+As Carbon's design and implementation begin to take shape, we'll inevitably
+encounter with plenty of bugs. Helping us triage, analyze, and address them is
+always a great way to get involved. See
+[open issues on GitHub](https://github.com/carbon-language/carbon-lang/issues).
```
Comment (chandlerc) [inline CONTRIBUTING.md]:
```suggestion
As Carbon's design and implementation begin to take shape, we'll inevitably
encounter plenty of bugs. Helping us triage, analyze, and address them is
always a great way to get involved. See
[open issues on GitHub](https://github.com/carbon-language/carbon-lang/issues).
```

File: README.md [hunk 2]
```diff
@@ -309,6 +309,21 @@ Past Carbon focused talks from the community:
 -   [Carbon Language: Syntax and trade-offs](https://www.youtube.com/watch?v=9Y2ivB8VaIs),
     Core C++ 2022
 
+## Contributing
```
Comment (chandlerc) [inline README.md]: What do you think about merging this with the next section?

```
## Contributing

We'd love to have folks join us and contribute to the project. Carbon is committed to a welcoming and inclusive environment where everyone can contribute.

<old list of bullet points>

You can also directly contribute to the design or implementation:

<your new list of bullet points>

You can refer to ... <rest of your new paragraph>
```

**Review by chandlerc — APPROVED** [review]
Sorry I got distracted and forgot about this for a week, thanks for working on getting this better surfaced in our docs! Some ideas inline.

**Review by chandlerc — CHANGES_REQUESTED** [review]
That was *supposed* to just be the "comment" button, but requesting changes now to try to reset the approve button... :sigh: sorry for the chaos!

File: README.md (hunk 2 above)
Comment (Pixep) [reply README.md]: I think that might be confusing to people who don't want to contribute, but just have a few questions, or want to stay up to date (i.e. a `#Join us` / stay in the loop / ...).

But I believe we can swap `#Join us` and `# Contributing` for a similar effect

**Comment by Pixep** [comment]
> That was _supposed_ to just be the "comment" button, but requesting changes now to try to reset the approve button... :sigh: sorry for the chaos!

No worries. Pushed a suggestion in that direction, lowered `contributing` to level 3 under `#Join us`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -84,24 +89,49 @@ eventually implemented. A good starting point is to read through the
...
+    [Explorer issues](https://github.com/carbon-language/carbon-lang/issues?q=is%3Aissue+is%3Aopen+label%3Aexplorer)),
+    and
+-   the [Carbon toolchain](/toolchain/) (see Carbon
+    [toolchain issues](https://github.com/carbon-language/carbon-lang/issues?q=is%3Aissue+is%3Aopen+label%3Atoolchain))
+
+The Carbon Explorer is generally simpler to contribute to, while the toolchain
+implements the compiler and tools needed for the language. Some issues are also
+marked as
 ["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue).
 These are intended to be a good place to start contributing.
```
Comment (chandlerc) [inline CONTRIBUTING.md]: Let's make this part really easy to find for folks:

```suggestion
The Carbon Explorer is generally simpler to contribute to, while the toolchain
implements the compiler and tools needed for the language. **Some issues are also
marked as
["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue)**.
These are intended to be a good place to start contributing.
```

File: README.md [hunk 4]
```diff
@@ -322,3 +323,19 @@ contribute.
...
+    feedback on design, new design proposal
+-   [Contribute to the language implementation](CONTRIBUTING.md#contributing-to-the-language-implementation)
+    -   [Carbon Explorer](/explorer/): bug report, bug fix, language feature
+        implementation
+    -   [Carbon Toolchain](/toolchain/), and project infrastructure
+
+You can refer to [`CONTRIBUTING`](CONTRIBUTING.md) for more information, check
+["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue),
+or join the `#contributing-help` channel on
+[Discord](https://discord.gg/ZjVdShJDAs).
```
Comment (chandlerc) [inline README.md]: Slight structure / formatting tweaks, WDYT?

```suggestion
You can **check out some
["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue)**,
or join the `#contributing-help` channel on
[Discord](https://discord.gg/ZjVdShJDAs). See our full
[`CONTRIBUTING`](CONTRIBUTING.md) documentation for more details.
```

**Review by chandlerc — APPROVED** [review]
LGTM w/ some formatting / structure tweaks below. Thanks!

File: README.md (hunk 4 above)
Comment (Pixep) [reply README.md]: Yep, works for me!

**Pull request merged**